docs/proposal/20201103-failure-domain.md

```diff
@@ -0,0 +1,377 @@
+# CAPV ControlPlane Failure Domain
```

Comment:
@yastij mdlint doesn't like the proposal template to have title after the status. So I'm adding this hack to put the line 4-20 within a code block, and move H1 to first line.